change/@fluentui-react-experiments-74f78e6d-e0d1-4f9d-9504-b17e6521c623.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "UnifiedPicker: Don't show picker after input is cleared",
+  "packageName": "@fluentui/react-experiments",
+  "email": "elvonspa@microsoft.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
don't need date on this one?

Reply:
apparently note. this file is auto generated, so it's whatever the fluent team wants to be there.